server/startup/migrations/v212.js

```diff
@@ -0,0 +1,21 @@
+import { MongoInternals } from 'meteor/mongo';
+
+import { Migrations } from '../../../app/migrations';
+
+Migrations.add({
+	version: 212,
+	async up() {
```

Comment:
migration cannot be `async` as it returns immediately and the runner is not awaiting the result.